graph/graph.go

```diff
@@ -62,22 +62,27 @@ func (g *Graph) Blocks(from, to *ssb.FeedRef) bool {
 	return math.IsInf(w.Weight(), 1)
 }
 
-func (g *Graph) BlockedList(from *ssb.FeedRef) map[librarian.Addr]struct{} {
+func (g *Graph) BlockedList(from *ssb.FeedRef) *ssb.StrFeedSet {
 	g.Mutex.Lock()
 	defer g.Mutex.Unlock()
+	blocked := ssb.NewFeedSet(0)
 	nFrom, has := g.lookup[from.StoredAddr()]
 	if !has {
-		return nil
+		return blocked
 	}
-	blocked := make(map[librarian.Addr]struct{})
 	edgs := g.From(nFrom.ID())
 	for edgs.Next() {
 		nTo := edgs.Node()
 		edg := g.Edge(nFrom.ID(), nTo.ID()).(contactEdge)
 
 		if math.IsInf(edg.Weight(), 1) {
 			ctNode := nTo.(*contactNode)
-			blocked[ctNode.feed.StoredAddr()] = struct{}{}
+			fr, err := ctNode.feed.FeedRef()
+			if err != nil {
+				panic(err)
```

Comment:
Looks drastic :|

Reply:
i know. the rational is, that only valid feeds are on the graph. I should comment this.